src/Client.php

```diff
@@ -166,25 +161,20 @@ public function __construct(Configuration $config, TransportInterface $transport
     /**
      * {@inheritdoc}
      */
-    public function getBreadcrumbsRecorder()
+    public function addBreadcrumb(Breadcrumb $breadcrumb, ?Scope $scope = null)
```

Comment:
Actually it's needed because of the need to check the configuration options. The problem is that I forgot to call it from the `Hub` instead of directly calling the `Scope`. Said that, given that the `Hub` knows about the client and the scope and that that method cannot work without the scope instead I think we can safely remove it. @HazAT WDYT?

Reply:
The way I see it, the client should become just a sender of events through the Transport... Maybe we should move the config elsewhere, or use it in the hub too?

Or maybe we can split it in two, with client/transport related stuff in one, and data/event related stuff in the other?

Reply:
So we leave it as it is right now, you are right, the client is basically nothing more than an API over a transport. The Client should be responsible for building together an event and delegate the sending to a transport.